app/mailers/references_header_builder.rb

```diff
@@ -22,6 +22,10 @@ def build_references_header(conversation, in_reply_to_message_id)
 
     references = references.compact.uniq
     fold_references_header(references)
+  rescue StandardError => e
+    Rails.logger.error("Error building references header for ##{conversation.id}: #{e.message}")
+    ChatwootExceptionTracker.new(e, account: conversation.account).capture_exception
```

Comment:
@muhsin-k added this check to ensure that email sending never breaks at all. References header is not critical to sending. We can remove this in the future